packages/mco/components/modals/app-manage-policies/parsers/application-set-parser.spec.tsx

```diff
@@ -345,6 +345,11 @@ jest.mock('@odf/mco/hooks/acm-safe-fetch', () => ({
   }),
 }));
 
+jest.mock('@openshift-console/dynamic-plugin-sdk', () => ({
+  ...jest.requireActual('@openshift-console/dynamic-plugin-sdk'),
+  useK8sWatchResource: jest.fn().mockReturnValue([[], false]),
```

Comment:
`loading` state should be `true` if mocking...
```suggestion
  useK8sWatchResource: jest.fn(() => [[], true, undefined]),
```

Reply:
done